content/en/platform/corda/5.2/deploying-operating/keys/_index.md

```diff
@@ -0,0 +1,196 @@
+---
+description: "Learn how to upload and rotate Corda 5 wrapping keys."
+title: "Managing Wrapping Keys"
+date: '2024-02-12'
+menu:
+  corda52:
+    identifier: corda52-cluster-wrapping-keys
+    parent: corda52-cluster
+    weight: 3025
+---
+
+# Managing Wrapping Keys
+
+As described in the [Architecture for Cluster Administrators]({{< relref "../../key-concepts/cluster-admin/_index.md#key-management" >}}) section, Corda keys are stored in the cluster and virtual node `Crypto` databases and encrypted at rest with wrapping keys.
+These managed wrapping keys are in turn wrapped by a master wrapping key.
```

Comment:
I don't think "managed wrapping keys" is a term end users will understand without more explanation.

How about:
These wrapping keys are created and managed automatically by Corda, hence we refer to them as managed wrapping keys. These managed wrapping keys are in turn encrypted with a master wrapping key and also stored in the cluster and virtual node `Crypto` databases.